packages/solaris/packer/scripts/solaris/postinstall.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/sh
+# postinstall script for Solaris VMs
+# Wazuh, Inc 2015
+
+# install wazuh dependencies & some important packages
```

Comment:
I prefer not to modify any of these files, since they have been completely migrated from wazuh-packages, and in principle they will not be needed, so let's leave them as they are and in the future maybe we can delete them completely.